### PR TITLE
refactor(tsc-wrapped): move tsc-wrapped to packages directory

### PR DESCRIPTION
**Pull request by chuckjaz**
## PR Type
What kind of change does this PR introduce?

```
[x] Refactoring (no functional changes, no api changes)
```

## What is the current behavior?

`tsc-wrapped` builds and is published out of the tools directory.

## What is the new behavior?

`tsc-wrapped` is built and is published out of the packages directory matching all of the other published packages.

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -410,10 +408,20 @@ fi
 if [[ ${BUILD_ALL} == true ]]; then
   rm -rf ./dist/packages
   if [[ ${BUNDLE} == true ]]; then
-    rm -rf ./dist/packages-dist
+    # Delete everything except tsc-wrapped
```
Comment (alexeagle) [inline build.sh]: is it actually doing that?

**Review by alexeagle — APPROVED** [review]
hard for me to tell if it's all the correct spots touched.
Maybe Jason should look too?

File: build.sh (hunk 1 above)
Comment (chuckjaz) [inline build.sh]: Need to remove the comment as I moved where `tsc-wrapped` is build instead.

**Review by mhevery — APPROVED** [review]

**Review by jasonaden — APPROVED** [review]
lgtm

**Review by vicb — APPROVED** [review]

File: packages/tsc-wrapped/package.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {
   "name": "@angular/tsc-wrapped",
-  "version": "4.3.0",
+  "version": "0.0.0-PLACEHOLDER",
```
Comment (IgorMinar) [inline packages/tsc-wrapped/package.json]: yay!!!!

**Review by IgorMinar — APPROVED** [review]
the currently hardcoded version needs to be changed to 0.0.0-PLACEHOLDER in compiler-cli/package.json

otherwise lgtm

**Comment by chuckjaz** [comment]
It turns out that changing the dependency in compiler-cli to a PLACEHOLDER causes the integration tests to fail as yarn cannot find the dependency. This works for peer dependencies since they are verified after all the packages are installed so we can just install the right version but since it is a normal dependencies yarn looks in the npm repository for it and cannot find it.

**Comment by IgorMinar** [comment]
discussed in person with @chuckjaz and it seems that the best option is to keep the tsc-wrapped version in compiler-cli pinned to simplify the integration test setup. the tsc-wrapped package will go away soon, so it doesn't make sense to spend too much time on making the integration test better in this way now.

**Comment by chuckjaz** [comment]
CARETAKER: do not squash the commits as the first commit moves the unmodified files.

**Pull request closed without merging**